src/PattyClient.js

```diff
@@ -188,9 +190,10 @@ class PattyClient {
     return this._retry(queryOptions.retries, queryOptions.retryDelay, () => {
       // adding a promise timeout because this is longer than supposed to on windows.
 
-      return this
-        ._httpPromise('POST', '/', query, queryOptions.timeout)
-        .timeout(queryOptions.timeout, 'TIMEOUT');
+      return Promise.race([
```

Comment:
emulating a `Bluebird.timeout(n)` with `Promise.race` of two promises, one of which rejects after `n` milliseconds.